test/css-stats.json

```diff
@@ -4,7 +4,7 @@
     "base64Length": 0,
     "redundantBodySelectors": 0,
     "redundantChildNodesSelectors": 0,
-    "colors": 39,
+    "colors": 41,
```

Comment:
Do we have some boundaries for these stats? Or what are they used for?

Reply:
Right now, no we don't — they’re just checked manually for any standout differences introduced in a PR. I find it useful for deciding where to refactor or cleanup too. Initially we thought of having the changes mapped over time, and even some kind of 'complexity' budget, but we didn’t get to it yet :)